build.sbt

```diff
@@ -315,6 +315,23 @@ lazy val docSettings = Seq(
   mdocExtraArguments := Seq("--no-link-hygiene")
 )
 
+lazy val docs_scala3Settings = docSettings ++ Seq(
+  Compile / scalacOptions ++= Seq("-no-indent"), // Fix for indent errors in mdoc
+  ScalaUnidoc / unidoc / scalacOptions ++= Seq(
+    "-external-mappings:" +
+      ".*scala/.*::scaladoc3::https://dotty.epfl.ch/api/," +
+      ".*java/.*::javadoc::https://docs.oracle.com/javase/8/docs/api/",
+    "-social-links:" +
+      "github::https://github.com/typelevel/cats," +
+      "gitter::https://gitter.im/typelevel/cats," +
+      "twitter::https://twitter.com/typelevel," +
+      "discord::https://discord.gg/hWd4eS244g",
+    "-Ygenerate-inkuire",
+    "-project-logo",
+    "docs/src/main/resources/microsite/img/cats-logo.png"
+  )
+)
+
```

Comment:
None of this is particularly specific to Cats actually :) we should explore upstreaming these into sbt-typelevel.